auth/user_test.go

```diff
@@ -90,6 +172,9 @@ func TestGetToken_BrowserFlow(t *testing.T) {
 			// Run the test.
 			userSession := UserSession{}
 			userSession.browser = &mockBrowser{}
+			userSession.refreshTokenSourceFunc = func(ctx context.Context, config *oauth2.Config, token *oauth2.Token) oauth2.TokenSource {
+				return &mockRefreshTokenSource{}
```

Comment:
Shouldn't we be asserting that this was called / the mock browser wasn't in the new test case you added?

Reply:
They return different token values, so we are able to know that the token we received came from mockRefreshTokenSource.